guacamole-ext/src/main/resources/org/apache/guacamole/protocols/rdp.json

```diff
@@ -98,6 +98,7 @@
                         "it-it-qwerty",
                         "ja-jp-qwerty",
                         "sv-se-qwerty",
+			"es-es-qwerty",
```

Comment:
Please maintain alphabetical order of the values listed (with the exception of "failsafe" and "de-de-qwertz", where we slipped up).